specs/v1.yaml

```diff
@@ -422,6 +422,441 @@ paths:
           $ref: "#/components/responses/NotFoundError"
         '500':
           $ref: "#/components/responses/InternalServerError"
+  /v1/webhooks:
+    get:
+      summary: List Webhooks
+      description: List Webhooks for a subscription
+      operationId: list-webhooks
+      security:
+        - auth0:
+          - "read:webhooks"
+      parameters:
+        - name: pageLimit
+          in: query
+          description: the number of results to return; defaults to 10, max is 100
+          schema:
+            type: integer
+            default: 10
+        - name: pageToken
+          in: query
+          description: |-
+            A token to use to retrieve the next page of results. This is useful for paginating over many pages of results. If set, all other arguments are expected to be kept the same as previous calls.
+          schema:
+            type: string
+      responses:
+        '200':
+          $ref: "#/components/responses/ListWebhooksResponse"
+        '400':
+          $ref: "#/components/responses/BadRequestError"
+        '401':
+          $ref: "#/components/responses/AuthenticationError"
+        '403':
+          $ref: "#/components/responses/ForbiddenError"
+        '500':
+          $ref: "#/components/responses/InternalServerError"
+    post:
+      summary: Create Webhook
+      description: |-
+        Create a Webhook object.
```

Comment:
this description is not very descriptive. we need to write these descriptions keeping in mind that the people consuming may not know what a webhook is and why i would create one. we should make sure the descriptions for all the APIs keep this principle in mind.

e.g.

> Creates a Webhook and registers the provided URL endpoint to receive HTTP POST webhook messages. The endpoint for the Webhook should be able to accept HTTP POST requests. Please see ....

Reply:
done